src/Tokenizer/CT.php

```diff
@@ -54,6 +54,8 @@ final class CT
     public const T_NAMED_ARGUMENT_COLON = 10033;
     public const T_FIRST_CLASS_CALLABLE = 10034;
     public const T_TYPE_INTERSECTION = 10035;
+    public const T_DISJUNCTIVE_NORMAL_FORM_TYPE_BRACE_OPEN = 10036;
+    public const T_DISJUNCTIVE_NORMAL_FORM_TYPE_BRACE_CLOSE = 10037;
```

Comment:
```suggestion
    public const T_TYPE_PARENTHESIS_OPEN = 10036;
    public const T_TYPE_PARENTHESIS_CLOSE = 10037;
```

See #5915.

Reply:
fine by me to call these `PARENTHESIS` , I would leave the DNF part in the name as these should not be confused by any other parenthesis

Reply:
> I would leave the DNF part in the name as these should not be confused by any other parenthesis

Isn't that why we have `TYPE` in name? I'm OK with leaving it like that though :)

Reply:
yeah for now we have only one kind typing in PHP which is using `(` `)`, but I wouldn't be surprised if we get more in the future ;)